sentry-samples/kmp-app/iosApp/iosApp/iOSApp.swift

```diff
@@ -3,15 +3,15 @@ import shared
 
 @main
 struct iOSApp: App {
-    let sentry = Sentry()
-
+    let sentry = Sentry.shared
```

Comment:
Singletons mapped via Kotlin should be accessed via `.shared` or `.companion` or `Companion.shared`: https://kotlinlang.org/docs/native-objc-interop.html#kotlin-singletons